python/README.md

```diff
@@ -22,6 +22,17 @@ objection store communication. Please file Github issue to request for critical
 openssl upgrade.
 
 
+Examples
+------------
+
+The examples folder shows how to use Rust API to manipulate Delta tables.
```

Comment:
This README file change I think should be in `rust/README.md` rather than the `python/` one.